src/v3/tutorial/middleware/README.md

```diff
@@ -0,0 +1,87 @@
+---
+description: This tutorial will show you how to use a middleware.
+full_title: Middleware Tutorial
+---
+
+# Middleware Tutorial
+
+This tutorial will show you how to use a middleware. A middleware is used when an application wants to provide a way for other applications to change its behavior.
```

Comment:
I think this might read better like this:

```
This tutorial will show you how to use middleware.

Applications can register middleware to provide features and change the behavior of other applications.
```